src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -595,6 +595,7 @@ std::shared_ptr<ov::ICompiledModel> Plugin::compile_model(const std::shared_ptr<
     const auto platform = _backends->getCompilationPlatform(localConfig.get<PLATFORM>(), localConfig.get<DEVICE_ID>());
     auto device = _backends->getDevice(localConfig.get<DEVICE_ID>());
     localConfig.update({{ov::intel_npu::platform.name(), platform}});
+    Logger::global().setLevel(localConfig.get<LOG_LEVEL>());
```

Comment:
Perhaps we could do that earlier to entry-point as there some above code could log too.
just that would need to check if LOG_LEVEL is give in properties

Reply:
Moved to entrance of the function and can see log from config update when we use LOG_TRACE now.